src/KBS.FauxApplication/SingleBusControl.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using KBS.Messages;
+using MassTransit;
+
+namespace KBS.FauxApplication
+{
+    /// <summary>
+    /// Holds the bus control singleton
+    /// </summary>
+    public abstract class SingleBusControl
+    {
+        private static readonly object busLock = new object();
+        private static volatile IBusControl busControl;
+
+        /// <summary>
+        /// Get the bus control singleton, it's already started
+        /// </summary>
+        protected IBusControl BusControl
```

Comment:
```suggestion
        protected IBusControl GetBusControl
```

Reply:
Is a property  not a method, convention is without 'Get'-